nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/tools/ScanTable.java

```diff
@@ -561,9 +562,18 @@ public void loadFromDataset() {
         fileTableFilter.resetFilter();
         String firstDatasetName = dataset.getFileName();
         if (firstDatasetName.length() > 0) {
-            FXMLController.getActiveController().openDataset(dataset.getFile(), false, true);
+            FXMLController controller = FXMLController.getActiveController();
+            controller.openDataset(dataset.getFile(), false, false);
+            DatasetBase datasetReloaded = chart.getDataset();
             List<Integer> rows = new ArrayList<>();
             rows.add(0);
+            // Load from Dataset assumes an arrayed dataset
+            datasetReloaded.setNFreqDims(datasetReloaded.getNDim() - 1);
+            if (datasetReloaded.getNDim() > 2) {
+                chart.getDisDimProperty().set(PolyChart.DISDIM.TwoD);
+            } else {
+                chart.getDisDimProperty().set(PolyChart.DISDIM.OneDX);
```

Comment:
currently this doesnt update the Traces/Contours dropdown in the spectrum status bar, but in NMR-5731, I have bound the values, so the status bar will automatically update the values so didn't worry about manually changing it here